scrapy_crawlera.py

```diff
@@ -11,6 +12,8 @@
 
 __version__ = '1.0.2'
 
+uuid4hex = re.compile('[0-9a-f]{12}4[0-9a-f]{3}[89ab][0-9a-f]{15}\Z', re.I)
```

Comment:
it is ok, `APIKEY` is uuid4.hex https://github.com/scrapinghub/dash/blob/develop/hdash/utils/apikeys.py#L25

But I think we don't need to do this verification here.